src/Ironclad.Client/UsersHttpClient.cs

```diff
@@ -34,7 +34,7 @@ public UsersHttpClient(string authority, HttpMessageHandler innerHandler = null)
 
         /// <inheritdoc />
         public Task<ResourceSet<UserSummary>> GetUserSummariesAsync(
-            string startsWith = default,
+            string startsWith,
```

Comment:
Wasn't the idea of this method to have all parameters optional? By removing `=default` you will make this parameter mandatory.

Reply:
The same applies to other methods.

Reply:
this can't be an optional parameter in our case, because we can't return all the clients and there are no methods for this